docs/topics/spiders.rst

```diff
@@ -145,6 +145,41 @@ scrapy.Spider
        :param kwargs: keyword arguments passed to the :meth:`__init__` method
        :type kwargs: dict
 
+   .. classmethod:: update_settings(settings)
+
+       The ``update_settings()`` method is used to modify the spider's settings
+       and is called during initialization of a spider instance.
+
+       It takes a :class:`~scrapy.settings.Settings` object as a parameter and
+       can add or updates the spider's configuration values. This method is a class method,
+       meaning that it is called on the :class:`~scrapy.Spider` class and allows all instances
+       of the spider to share the same configuration.
+
+       One of the main advantages of ``update_settings()`` is that it allows
+       you to dynamically add, remove or change settings based on spider arguments
+       or other external factors.
+
+       For example, suppose a spider needs update :setting:`FEEDS`:
```

Comment:
```suggestion
       For example, suppose a spider needs to modify :setting:`FEEDS`:
```

Besides the missing “to” typo, I think “modify” here (or “edit”, or “change”) is a slightly better word choice here, because I think the point is that, unlike `custom_settings` that only let you completely override a setting value from e.g. the command line, here you can keep parts of the command-line value while changing other parts.